app/components/community_activity_component.rb

```diff
@@ -26,4 +26,8 @@ def reopen_button_text
     return "Add more evidence" if achievement_rejected?
     @achievement&.evidence.present? ? "Continue editing" : "Submit evidence"
   end
+
+  def minimum_character_requirement
+    @activity.programmes.map(&:minimum_character_required_community_evidence).max
```

Comment:
Just as an FYI, there is method called collect that I used when extracting data from an array of objects. It makes no difference which you use, as collect is just an alias for map. I use it as I think its clearer I am just 'collecting' the values, not doing something to them.